templates/README.md

```diff
@@ -0,0 +1,14 @@
+### Create Secret in Kubernetes
+
+```bash
+    kubectl create -f templates/secret.yaml
+```
+
+### Create Service Account, Role, Rolebinding and Cronjob in Kubernetes
+
+```bash
+    kubectl create -f templates/cronjob.yaml
+```
+
+
+>  More information is available at [AWS IAM key rotate tool](https://github.com/nuxeo-cloud/aws-iam-credential-rotate)
```

Comment:
Could you please add a title starting with a `#` to have valid Markdown?
Something like:
```
# Kubernetes Manifest Files
```
And a small paragraph below the title explaining that these files are deployed in the `platform` namespace.

Then, you can use `##` for subtitles instead of `###`.

Note: the `markdownlint` VSCode plugin is nice to have warnings about the Markdown syntax. For instance, there I have:
```
MD041/first-line-heading/first-line-h1: First line in file should be a top level headingmarkdownlint(MD041)
```

Reply:
Also, in "More information...", there's an extra white space after `>` and a missing `.` at the end of the sentence.

Reply:
hello @ataillefer ,
remark have ben take to account